src/plugins/image_embeddable/public/components/image_editor/image_editor_flyout.tsx

```diff
@@ -56,13 +56,14 @@ export interface ImageEditorFlyoutProps {
   onCancel: () => void;
   onSave: (imageConfig: ImageConfig) => void;
   initialImageConfig?: ImageConfig;
-  validateUrl: ValidateUrlFn;
```

Comment:
Since `validateUrl` is provided as part of the `ImageViewerContext`, we don't need to also send it as a prop to `ImageEditorFlyout` - so, I removed it.